### PR TITLE
#33 Add models based on db schema

### PR DESCRIPTION
**Pull request by andpii**

### CODE REVIEW

File: backend/Tasque.Core.Common/Entities/Abstract/BaseEntity.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿namespace Tasque.Core.Common.Entities.Abstract;
+
+public class BaseEntity
```
Comment (ihor-il) [inline backend/Tasque.Core.Common/Entities/Abstract/BaseEntity.cs]: maybe abstract class?

File: backend/Tasque.Core.Common/Entities/Task.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    public string Name { get; set; } = "";
+    public string Description { get; set; } = "";
+    public string Summary { get; set; } = "";
+
+    public TaskState State { get; set; }
+    public TaskType Type { get; set; }
+    public TaskPriority Priority { get; set; }
+
+    public DateTime Deadline { get; set; }
+    public DateTime FinishedAt { get; set; }
```
Comment (ihor-il) [inline backend/Tasque.Core.Common/Entities/Task.cs]: Is FinishedAt nullable?

File: backend/Tasque.Core.Common/Entities/Abstract/BaseEntity.cs (hunk 1 above)
Comment (andpii) [reply backend/Tasque.Core.Common/Entities/Abstract/BaseEntity.cs]: Done

File: backend/Tasque.Core.Common/Entities/Task.cs (hunk 2 above)
Comment (andpii) [reply backend/Tasque.Core.Common/Entities/Task.cs]: Fixed. It is

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by FeloniukM — APPROVED** [review]

**Pull request merged**